packages/TextEditor/src/TextEditor.js

```diff
@@ -1,5 +1,79 @@
 import React from "react";
+import PropTypes from "prop-types";
+import { Editor } from "@tinymce/tinymce-react";
+import { kind as kindType } from "./types";
+import * as simpleConfig from "./config/simple.config";
 
-export default function TextEditor() {
-  return <>Text editor</>;
+function getPlugins(kind) {
```

Comment:
I suspect this would be one of the first things people want to override.  Should they be able to?

Reply:
yeap it will possible in an incoming PR.